App/main.swift

```diff
@@ -1,9 +1,9 @@
 import Vapor
 import VaporStencil
 
-//set the stencil renderer
-//for all .stencil files
-View.renderers[".stencil"] = StencilRenderer()
+let application = Application([
+	VaporStencil.Provider // Adds support for stencil rendering for all .stencil views
+])
 
 Route.get("/") { request in
```

Comment:
We will need to change `Route` to `application` or perhaps `app` is more concise.